LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2016-2022, QIIME 2 development team.
+Copyright (c) 2022, QIIME 2 development team.
```

Comment:
@thermokarst I figured we should change the year since we are essentially writing a completely new plugin - the name happens to be the same (hence why we didn't just create an entirely new repo), but nothing else really will be. But I can revert the year back if that makes more sense!

Reply:
I think personally I would prefer to see this reverted - many folks have contributed to "v1" of this plugin over the years, let's not just sweep that under the rug.

Reply:
Sounds good, can do!